PAWI_HS18_Projektdokumentation.tex

```diff
@@ -131,12 +131,34 @@ \section{Aufgabenstellung und Zielsetzung}
 \chapter{Grundlagen}
 \label{ch:StandDerForschung}
 
-% TODO some Text about what we will do in this chapter...
+Dieses Kapitel bietet einen Übersicht über die Entwicklung der Augmented Reality und deren Technologien. Es führt weiterhin wichtige Konzepte und Begriffe dieser Arbeit ein. Es soll dem Leser einen Startpunkt in dieses spannende Thema bieten und in unser Projekt einführen.
```

Comment:
"in unser Projekt" durch "in das Projekt" ersetzten.
Evt. "spannende" löschen, da es "nicht sehr" Wisenschaftlich ist.